opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -43,4 +43,35 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successful export request.
+  //
+  // If the request is only partially accepted
+  // (i.e. when the server accepts only parts of the data and rejects the rest)
+  // the server MUST initialize the `partial_success` field and MUST
+  // set the `rejected_log_records` with the number of log records it rejected.
+  //
+  // The server MUST NOT set the `partial_success` field when the
+  // request was accepted in its entirety.
+  ExportLogsPartialSuccess partial_success = 1;
+}
+
+message ExportLogsPartialSuccess {
+  // The number of rejected log records.
+  //
+  // If the request is only partially accepted, the server MUST set
+  // the `rejected_log_records` field.
+  // 
+  // Given the server should only initialize the `partial_success` field
```

Comment:
```suggestion
  // Given the server MUST only initialize the `partial_success` field
```
I think you can just use the same level as in line 53 above.

Reply:
:+1: and the same change in the two appearances below.

Reply:
@jmacd  you mean the next "should" as in here change to MUST?

```
// A developer-facing human-readable error message in English. It should..
// explain why the server rejected parts of the data,
```

I ask because it's different from what I have currently in the related specification PR. Do we want to enforce servers to _always_ populate the message?

Reply:
Changed parts of it in [32ab10e](https://github.com/open-telemetry/opentelemetry-proto/pull/414/commits/32ab10ec580906780e6cedb47f362c8d35be83f1).

Reply:
Sorry for being unclear. Your changes are good, I just meant to apply the same suggestion for traces and metrics sections.